modules/ossm-install-kiali-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-kiali-operator_{context}"]
+= Installing the {KialiProduct}
+:context: ossm-install-kiali-operator
+
+// TP 1 Content
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+// Kiali install content for TP1 is the same as install content for 2.x.
+
+The following steps show how to install the {KialiProduct}.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
```

Comment:
Nit - is there a reason that Operators is plural here? It sounds a bit off saying "the Operators" but it not being clear which specific Operators are being referred to here. I would choose one of the following changes:

- Removing "the" if this is a general warning about not installing community versions of _any_ Operator.
- Replace "the" with "any" if this is meant to be a lot more proscriptive and tell users to, like, _never_ bother with community operators
- Making Operators singular if this is really only meant to reference the Kiali Operator

Reply:
In 2.x, all of the operators were installed as a group, and the user has the option of selecting a Red Hat version or a community version. This was copy/pasted from existing 2.x docs and I did not catch that it says "Operators." Thank you!